scripts/exgdas_global_marine_analysis_bmat.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+################################################################################
+####  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exgdas_global_marine_analysis_run.sh
+# Script description:  Runs the global marine analysis with SOCA
+#
+# Author: Guillaume Vernieres        Org: NCEP/EMC     Date: 2022-04-24
+#
+# Abstract: This script makes a global model ocean sea-ice analysis using SOCA
```

Comment:
update the description maybe.
That scripts does the following:
- generate DA grid
- computes diagonal of B based on the background
- creates the bump correlation operators
and I think that's it